src/helpers/get-insights-anonymous-user-token.ts

```diff
@@ -0,0 +1,21 @@
+export const ANONYMOUS_TOKEN_COOKIE_KEY = '_ALGOLIA';
+
+function getCookie(name: string): string {
+  const prefix = `${name}=`;
+  const cookies = document.cookie.split(';');
+  for (let i = 0; i < cookies.length; i++) {
+    let cookie = cookies[i];
+    while (cookie.charAt(0) === ' ') {
+      cookie = cookie.substring(1);
+    }
+    if (cookie.indexOf(prefix) === 0) {
+      return cookie.substring(prefix.length, cookie.length);
+    }
+  }
+  return '';
```

Comment:
should we not return undefined here instead?

Reply:
I thought about returning undefined first, but we have this ESlint rule `consistent-return`.
I can override it, would you recommend it in this case?

Reply:
If you explicitly return `undefined` it should work.

Reply:
awesome! thanks!

Reply:
fixed aee4083